pkg/cluster/config.go

```diff
@@ -1,25 +1,39 @@
-package clusternetwork
+package cluster
 
 import (
 	"context"
 	"fmt"
 	"net"
+	"strings"
 
+	oapi "github.com/openshift/api/config/v1"
 	configclient "github.com/openshift/client-go/config/clientset/versioned"
 	operatorv1 "github.com/openshift/client-go/operator/clientset/versioned/typed/operator/v1"
 	"github.com/pkg/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
 )
 
-const ovnKubernetesNetwork = "OVNKubernetes"
+const (
+	ovnKubernetesNetwork = "OVNKubernetes"
+	// baseK8sVersion specifies the base k8s version supported by the operator. (For eg. All versions in the format
+	// 1.19.x are supported for baseK8sVersion 1.18)
+	baseK8sVersion = "1.19"
+)
 
-// ClusterNetworkConfig interface contains methods to validate network configuration of a cluster
-type ClusterNetworkConfig interface {
+// NetworkConfig interface contains methods to validate network configuration of a cluster
+type NetworkConfig interface {
 	Validate() error
 	GetServiceCIDR() (string, error)
 	VXLANPort() string
 }
 
+type Config interface {
```

Comment:
doc missing for interface, can this renamed to a specific config like clusterConfig?